textile-utils-api-v1/src/main/java/net/textilemc/textile/utils/Pair.java

```diff
@@ -0,0 +1,16 @@
+package net.textilemc.textile.utils;
+
+public class Pair<T, M> {
```

Comment:
Doesn’t guava have Pair?

Reply:
If not this should probably go in api base

Reply:
Guava isn't an included library, so we can decide to keep our own, or use Guava here.

Reply:
Loader uses guava iirc

Reply:
Huh? if it does, then could you ready up a commit for using that Pair instead?